exemplary_client/README.md

```diff
@@ -0,0 +1,108 @@
+# Python Exemplary Client
+
+This is an exemplary command-line tool to simplify the process of running a
+Batch SQL Translation job using the Google Cloud Bigquery Migration Python
+client package.
+
+## Installation
+
+Download the entire directory from the [exemplary_client](https://github.com/xiashang0624/dwh-migration-tools/tree/python-client/exemplary_client) on GitHub and
+install the Python packages listed in the [requirements.txt](https://github.com/xiashang0624/dwh-migration-tools/tree/python-client/exemplary_client/requirements.txt) file.
+
+## User Manual
+
+Open the [config.yaml](https://github.com/xiashang0624/dwh-migration-tools/tree/python-client/exemplary_client/config.yaml) file and fill all the required fields. If you are a first
+time user who just wants to give it a try, we recommend to create a new [GCP
+project](https://pantheon.corp.google.com/) and put the project_number in the config. You can find the project
+number here.
+
+<div style="text-align:center">
+    <img src="img/example_gcp_project_number.png" alt="project_number" width="400"/>
+</div>
+
+If you want to use an existing project, make sure you have all the required [IAM
+permissions](https://cloud.google.com/bigquery/docs/batch-sql-translator#required_permissions).
+
+## input_directory
+
+The input directory is supposed to contain files with pure SQL statements (comments
+are OK). The file type can be in any format like .txt or .sql.
+
+TODO: add instruction about how to use the metadata dumper outputs as inputs here.
+
+## output_directory
+
+In the config, specify a local directory to store the outputs of the translation job. 
+Every input SQL file will have a corresponding output file under the same name in 
+the output directory.
+
+## Run a translation job
+
+Simply run the following commands in Python3:
+
+```
+python main.py
+```
+
+### [Optional] gcloud login and authentication
+
+Note that the program will first validate the login and credential status of
+gcloud. If the validation steps failed, the program will run the following two
+commands automatically and bring up a page on browser for your aggreement of
+using one of you google accounts.
+
+However, it's recommended that you run these two commands on the terminal first:
+
+Log in to gcloud:
+
+```
+gcloud auth login
```

Comment:
Add a link to documentation on how to install `gcloud`?

Reply:
Done.